.latexmkrc

```diff
@@ -0,0 +1,3 @@
+$pdf_previewer = 'open -a Skim';
+$pdflatex = 'pdflatex -synctex=1 -interaction=nonstopmode';
+@generated_exts = (@generated_exts, 'synctex.gz');
```

Comment:
I had to create this file in order to get compilation to work properly. I'm curious if you have something similar on your end.